src/core/lib/decorators/method.ts

```diff
@@ -111,7 +111,7 @@ function helperForRoutes(httpVerb: HttpDecorator, path?: string | RegExp): Metho
         } else if (path instanceof RegExp) {
             newPath = addForwardSlashToFrontOfRegex(path);
         } else { // assert (path instanceof string)
-            newPath = '/' + path;
+            newPath = path.charAt(0) === '/' ? path : '/' + path;
```

Comment:
After I changed this line, I decided to check how NestJS deals with paths under the hood (as they support forward slashes) and they do pretty much the same thing:

https://github.com/nestjs/nest/blob/489d3299c0b2d3ef15c5dfda9641313599b48a55/packages/common/utils/shared.utils.ts#L23-L24